test/fixtures/gulpfiles/gulpfile.js

```diff
@@ -20,6 +20,15 @@ function errorFunction() {
 function anon(cb) {
   cb();
 }
+
+function notCompleting1() {
```

Comment:
Do we need 2 non-completing tasks?

Reply:
I created 2 tasks to also test that the log message is rendered correctly with multiple non-completing tasks. Should I remove the second task?

Reply:
That's fine.  I'm not partial either way.